vllm/config.py

```diff
@@ -1338,6 +1338,7 @@ class ParallelConfig:
     # will be determined based on the platform.
     worker_cls: str = "auto"
     sd_worker_cls: str = "auto"
+    scheduler_cls: str = "vllm.core.scheduler.Scheduler"
```

Comment:
I think you can make the type either a string (qualname) or a type directly, similar to `distributed_executor_backend` that can directly be a class object. then you can easily test it, similar to https://github.com/vllm-project/vllm/blob/main/tests/engine/test_executor.py